_posts/en/newsletters/2024-06-07-newsletter.md

```diff
@@ -0,0 +1,243 @@
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 24.05rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- Bitcoin Core 27.1rc1 FIXME:harding
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Core Lightning #7252][] changes the behavior of `lightningd` to ignore the
+  `ignore_fee_limits` setting during a cooperative channel closure. This fixes
+  an issue where a Core Lightning (CLN) channel opener node overpays fees when
+  the counterparty is a LDK node. In this scenario, when the LDK non-opener node
+  (Alice) initiates a cooperative channel closure and begins fee negotiation,
+  the CLN opener node (Bob) responds that the fee can be anything between
+  `min_sats` and `max_channel_size` due to the `ignore_fee_limits` setting. Bob
+  picks the upper bound (contrary to the BOLTs specification), and Alice
+  accepts, resulting in Alice broadcasting a transaction with considerably
+  overpaid fees.
+
+- [LDK #2931][] enhances the logging during path finding to include additional
+  data about direct channels such as whether they’re missing, their minimum
+  [HTLC][topic htlc] amount, and their maximum HTLC amount. The added logging
+  aims to better troubleshoot routing issues by providing visibility into the
+  available liquidity and limitations on each channel.
+
+- [Rust Bitcoin #2644][] adds HKDF (HMAC (Hash-based Message Authentication
+  Code) Extract-and-Expand Key Derivation Function) to the `bitcoin_hashes`
+  component to implement BIP324 in Rust Bitcoin. HKDK is used to derive
+  cryptographic keys from a source of keying material in a secure and
+  standardized way. [BIP324][topic v2 p2p transport] (also known as v2 P2P
+  transport) is a proposal to allow Bitcoin nodes to communicate with each other
+  over encrypted connections (enabled by default in Bitcoin Core). 
+
+- [BIPs #1541][] adds BIP431 to propose [Topologically Restricted Until
+  Confirmation (TRUC)][topic v3 transaction relay] (v3 transactions) which are a
+  subset of standard transactions with additional rules designed to allow
+  transaction replacement while minimizing the cost of overcoming
+  [transaction-pinning][topic transaction pinning] attacks. See Newsletters
+  [#289][news289 v3], [#296][news296 v3] and [#301][news301 v3] for more on v3
```

Comment:
@Gustavojfe Thanks for including these links to previous discussions!  In cases where we have a page dedicated to a topic (e.g. v3 tx relay), we don't normally need to link to previously mentions of that topic.  One of our main motivations for adding topic pages was so that we didn't need to reintroduce a common topic each time we mentioned it and didn't need to manually insert links to each previous time it was mentioned.

Reply:
understood, thanks for explaining.